_dev/src/store/modules/accounts/actions.ts

```diff
@@ -700,4 +701,15 @@ export default {
       console.error(error);
     }
   },
+
+  async [ActionsTypes.REQUEST_ACCOUNTS_DETAILS]({
+    dispatch,
+  }) {
+    try {
+      await dispatch(ActionsTypes.REQUEST_GOOGLE_ACCOUNT_DETAILS);
+      await dispatch(`productFeed/${ActionsTypesProductFeed.GET_PRODUCT_FEED_SETTINGS}`, null, {root: true});
+    } catch (error) {
+      console.error(error);
+    }
```

Comment:
Why this `try {} catch {}`? All errors have been caught in both actions to return a default value